src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -697,6 +718,65 @@ public int updateMTimeForRecord(String recordId) throws Exception {
     }
     
     
+    
+      /**
+       * Create a new entry in the mapping table. The kalturaid can be null and will be updated by a job later.
+       * 
+       * @param mapping The mapping.id must not be null. The kalturaid can be null-
```

Comment:
the param is called mappingDto. Also, the mapping id, is that the referenceId?

Reply:
fixed.